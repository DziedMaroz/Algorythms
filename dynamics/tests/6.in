3143987552640
443
7910705330
77071
64283338367
80
075727239415
8
2560600566601
5
39136855
7874013287769
9179
030375704
4863012
03
7141011
54384344641
80
2924682852950
590657796777
39268
8
601176231
8788532893
6722440765
791
46
61905814
6988
5986
3597595538721
38
748271884
8865705
0638682868909
35875
28323210
7
818972276350
7165
514565
8
703
227201
47341755
077850
5955037616234
0872307778
315752
05732112676
27
803
15
803397893697
107244
0
1485
67898032
132537635
675485046
0933345417534
5919
5
0997412975
043
076
83
1780
7735972173
50147
47266500
59478403841
47652554666
650357232
09756
58574
1322747882
9427386810
4205497644
3290
18757116019
824
98797205927
0468965639
812
49450995486
80216135111
72
514321
524847740192
713
2867558
6502
72608935
71896636014
34088458
6278
046091274
80
493
195283894834
4997
8
870294702905
71
2
0211
5420496110
10982476
09121
60547746
2483190
2271730764
655311
515
1474287
3228911059
5
190366229912
852216652
34171804
80975046313
69877665
4382720
74
5967822533
0074723
4726695
3116396861
2072806487759
5638769292486
7150500416
645388124379
79339106
8832814
640531492
12922769869
34601594811
94075476160
9984945521870
5353406300
8
5291896
7945
341827
94513
6630045224360
74593835964
3151552841082
67736
102
1034766
059308
87594
7376217
4581
29501023181
345075801
800
11558
37847567606
7531823649
09007363
5696810250
7124
472223748
0
679239
16338
8752992361
44124
893595521
700231
7022
08080
15048361
065097
889658018
9026
35
1401331350489
24606891362
72156255529
8301
516141
4602
07
19564817221
8
240409
3653343076
0416896
1490616702061
848341087429
81944758684
4
944034
4856
654322367
728152408762
68
230
0760619249393
46812500
54033065
1
04
8689
8201477
977462541387
2590026746
58
97017745454
08477
3663639301406
55550
548980141
126374134048
76900
5
843543
58763116452
46927
4141920360324
27885834934
278468372780
97
7525
39219960404
540280037848
927080
99336
096
640393050
13811
22928635485
495191027
1386125362456
5043048739
69
983930
72008960
086
64867663
48992168
3
663851
80554012
67983
72086550
58280978
6264639
39886270590
43915
78
195
695334
46846
8
264596203
3519800717
4400892386337
84210854326
09
47515343699
688
18
2575
36
9814882
514872992468
50921
9366601099932
55
6128281664484
32992875
187433174213
580265751
39448485
620450570504
954466947
6292036
844391690
64066699866
2
287359674
73
78
72
96821049045
8
8
189
429
2
481999418
747651
03863306
78881997942
156529105688
17503038
98
12495
48808
80205184
55329110926
812585965805
835454472
49520286
057
7620871
7799628050457
07937
9074
315
55
766977
9
251
9031013
873821947
8699
81
9365821262245
8257362180
288
0259749
7181386370
102198
81760
64
08433
9492
6534293806
3
7159
090313174
91114906
6
820590
53235582138
129
5229
566
4270830216924
2
5928503
547606964
8998654
6252895994
7714396272875
2800
1687
87845
0
61887607273
3351045215903
83770874
311462236
395197
30182
05742171
0699081
32120240309
3698
5436
813
889959033
86
2
554832
19
94241
159856
340092
65933
8967483006674
6
9
714793583982
9
806889576
664885
8
4
6396031
628991485859
0797
879648670
16268192
2719384337
0568025
62413
58344
90776
3011179
617
72199600
128362
6
1629166096797
11627
6077
09159
8682115881286
364167959784
4943
2328399
4
668501
371608875
6238117053
4461
39213636804
605284
81
72570260876
522335993140
85797
43430213459
4
580
275939763437
064317
31
4079580
034037
5290028448757
5865182063
26
200232
384617
4
32689
670703
6703951
025260
73
715
86159444557
0678
1988
79
341517988
4363739
