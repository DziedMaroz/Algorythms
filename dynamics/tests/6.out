9
31 4 3 98 7 55 2 64 0
